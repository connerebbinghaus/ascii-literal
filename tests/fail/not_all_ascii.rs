const ASCII: &ascii::AsciiStr = ascii_literal::ascii_literal!("This is not ASCII! ðŸ˜¢");

fn main(){}