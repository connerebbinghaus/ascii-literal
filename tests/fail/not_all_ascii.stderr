error: String is not valid ascii!
 --> $DIR/not_all_ascii.rs:1:63
  |
1 | const ASCII: &ascii::AsciiStr = ascii_literal::ascii_literal!("This is not ASCII! ðŸ˜¢");
  |                                                               ^^^^^^^^^^^^^^^^^^^^^^^
